src/UserContainer.hpp

```diff
@@ -0,0 +1,27 @@
+
+#include <vector>
+#include <optional>
+
+#include "User.hpp"
+
+class UserContainer
+{
+private:
+    std::vector<User> users_;
+    
+public:
+    UserContainer() = default;
+
+    ~UserContainer() = default;
+
+    std::stringstream  showAll();
+
+    void createUser(const User & user);
```

Comment:
I don't like this name. It says createUser, but it takes already created user as an argument. I suggest to change the name to put/emplace/pushUser (your choice).

Reply:
it's not my name :)
I'm just cleaning here.

Reply:
So please create an Issue so I can formally change the name of the method

Reply:
I suggest "add" because we add user to the database.